strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/artifact/coordinates/PypiArtifactCoordinates.java

```diff
@@ -115,6 +115,17 @@ public PypiArtifactCoordinates(String distribution,
         setPackaging(packaging);
     }
 
+    public PypiArtifactCoordinates()
+    {
+        this(DISTRIBUTION,
+                VERSION,
+                "",
+                LANGUAGE_IMPLEMENTATION_VERSION,
+                ABI,
+                PLATFORM,
+                WHEEL_EXTENSION);
```

Comment:
Cosmetic: Could you please align chunked paramters to be one per line, aligned by the first parameter like this:
```
doFoo(param1,
      param2,
      param3);
```
For more details, please check our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html).